executor/executor.cc

```diff
@@ -620,6 +621,7 @@ void parse_env_flags(uint64 flags)
 	flag_vhci_injection = flags & (1 << 12);
 	flag_wifi = flags & (1 << 13);
 	flag_delay_kcov_mmap = flags & (1 << 14);
+	flag_nic_vf = flags & (1 << 15);
```

Comment:
Is it used? I don't see where.
If it's not used remove this and pkg/ipc.FlagEnableNicVF

Reply:
Left this flag. It is now checked instead of lspci output.